apps/src/applab/AppLabView.jsx

```diff
@@ -1,10 +1,19 @@
 /** @file Top-level view for App Lab */
 'use strict';
 
+var _ = require('lodash');
```

Comment:
There are also several of these outside this PR! :fearful: i.e. https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/calc/calc.js#L36